src/components/playback/playbackmanager.js

```diff
@@ -3163,82 +3153,82 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
                 return player.playbackStartTime();
             }
 
-            var streamInfo = getPlayerData(player).streamInfo;
+            const streamInfo = getPlayerData(player).streamInfo;
             return streamInfo ? streamInfo.playbackStartTimeTicks : null;
         };
 
         if (apphost.supports('remotecontrol')) {
-            require(['serverNotifications'], function (serverNotifications) {
+            import('serverNotifications').then(({ default: serverNotifications }) => {
                 events.on(serverNotifications, 'ServerShuttingDown', self.setDefaultPlayerActive.bind(self));
                 events.on(serverNotifications, 'ServerRestarting', self.setDefaultPlayerActive.bind(self));
             });
         }
     }
 
-    PlaybackManager.prototype.getCurrentPlayer = function () {
+    getCurrentPlayer() {
         return this._currentPlayer;
-    };
+    }
 
-    PlaybackManager.prototype.currentTime = function (player) {
+    currentTime(player) {
         player = player || this._currentPlayer;
         if (player && !enableLocalPlaylistManagement(player) && !player.isLocalPlayer) {
             return player.currentTime();
         }
 
         return this.getCurrentTicks(player);
-    };
+    }
 
-    PlaybackManager.prototype.nextItem = function (player) {
+    nextItem(player) {
         player = player || this._currentPlayer;
 
         if (player && !enableLocalPlaylistManagement(player)) {
             return player.nextItem();
         }
 
-        var nextItem = this._playQueueManager.getNextItemInfo();
+        const nextItem = this._playQueueManager.getNextItemInfo();
 
         if (!nextItem || !nextItem.item) {
             return Promise.reject();
         }
 
-        var apiClient = connectionManager.getApiClient(nextItem.item.ServerId);
+        const apiClient = connectionManager.getApiClient(nextItem.item.ServerId);
         return apiClient.getItem(apiClient.getCurrentUserId(), nextItem.item.Id);
-    };
+    }
 
-    PlaybackManager.prototype.canQueue = function (item) {
+    canQueue(item) {
         if (item.Type === 'MusicAlbum' || item.Type === 'MusicArtist' || item.Type === 'MusicGenre') {
             return this.canQueueMediaType('Audio');
         }
         return this.canQueueMediaType(item.MediaType);
-    };
+    }
 
-    PlaybackManager.prototype.canQueueMediaType = function (mediaType) {
+    canQueueMediaType(mediaType) {
         if (this._currentPlayer) {
             return this._currentPlayer.canPlayMediaType(mediaType);
         }
 
         return false;
-    };
+    }
 
-    PlaybackManager.prototype.isMuted = function (player) {
+    isMuted(player) {
```

Comment:
```suggestion
    isMuted(player = this._currentPlayer) {
```

Can we have all the functions that rely on the ``player`` object to have a default one? This will simplify a lot the use of those functions. See [here](https://github.com/jellyfin/jellyfin-web/pull/1430/files#diff-8cc2cabd6a4cde48b0ac68b500572723R3947). With the default argument, the call to getQueueShuffleMode() can be done in ``nowplaying``, no need to do ``getQueueShuffleMode(player)``.

Be careful because that PR caused some problems to other components because they passed wrongly ``player`` as ``null`` (something that wouldn't have happened in the first place if playbackManager had a more logical design, with those defaults, in the first place). **Default arguments are used if ``undefined`` is passed, but not ``null``**.

Reply:
This is not blocking or priority atm of course, I can do it in another PR as soon as I get some free time. But it's something nice to have.

Thank you very much as well for all the effort you're putting into ES6 migrations! You're doing it quick and fast! :P

Reply:
Thanks for reviewing, I have added the defaults, and tested as best as I can, it all seems to be working as intended.

Reply:
> Thank you very much as well for all the effort you're putting into ES6 migrations! You're doing it quick and fast! :P

My pleasure, onwards and upwards to Vue!